examples/openapi3/basicauth/app.py

```diff
@@ -4,29 +4,16 @@
 '''
 
 import connexion
-from connexion.decorators.security import validate_scope
-from connexion.exceptions import OAuthScopeProblem
 
+PASSWD={
+    'admin': 'secret',
+    'foo': 'bar'
+}
 
-def basic_auth(username, password, required_scopes=None):
-    if username == 'admin' and password == 'secret':
-        info = {'sub': 'admin', 'scope': 'secret'}
-    elif username == 'foo' and password == 'bar':
-        info = {'sub': 'user1', 'scope': ''}
-    else:
-        # optional: raise exception for custom error response
-        return None
-
-    # optional
-    if required_scopes is not None and not validate_scope(required_scopes, info['scope']):
-        raise OAuthScopeProblem(
-                description='Provided user doesn\'t have the required access rights',
-                required_scopes=required_scopes,
-                token_scopes=info['scope']
-            )
-
-    return info
-
+def basic_auth(username, password, required_scopes=None) -> dict:
```

Comment:
You don't have to typehint the return type here, but if you want to, it should be `typing.Optional[dict]` as it's possible to return `None`